ui/src/views/network/CreateIsolatedNetworkForm.vue

```diff
@@ -344,11 +344,13 @@ export default {
       var params = {
         zoneid: this.selectedZone.id,
         guestiptype: 'Isolated',
-        supportedServices: 'SourceNat',
         state: 'Enabled'
       }
-      if (isAdminOrDomainAdmin() && this.selectedDomain.id !== '-1') { // domain is visible only for admins
-        params.domainid = this.selectedDomain.id
+      if (isAdminOrDomainAdmin()) {
+        params.supportedServices = 'SourceNat'
```

Comment:
@utchoang this is applicable for normal users as they can only create isolated networks with source nat service.

Reply:
@sureshanaparti Does this mean that this change is not correct?

Reply:
Does it mean for normal users and admin can only add isolated networks with source nat service?

Reply:
normal users can add isolated networks with only source nat service

Reply:
Maybe I'll temporarily draft this PR and wait for discussion?

Reply:
Hi @utchoang Can you update the changes to enable only admin to view the network offerings without sourcenat. Users can see offerings with sourcenat enabled only.

Reply:
@utchoang Are the changes done, and is this PR ready?